packages/EasyServerless/bundle/EasyServerlessBundle.php

```diff
@@ -32,6 +33,10 @@ public function loadExtension(array $config, ContainerConfigurator $container, C
             ->set(ConfigParam::AssetsSeparateDomainEnabled->value, $config['assets_separate_domain']['enabled'])
             ->set(ConfigParam::AssetsSeparateDomainUrl->value, $config['assets_separate_domain']['url']);
 
+        if (\class_exists(EasyAdminBundle::class)) {
```

Comment:
In v6, Aleksey introduced a better way of detecting a bundle installed ([see the example](https://github.com/eonx-com/easy-monorepo/blob/master/packages/EasyActivity/bundle/EasyActivityBundle.php#L74)). Please check if it can be used here.